tests/migration_test.go

```diff
@@ -1266,6 +1266,45 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 			})
 		})
 
+		Context("migration to nonroot", func() {
+
+			BeforeEach(func() {
+				if !checks.HasFeature(virtconfig.NonRoot) {
+					Skip("Test specific to NonRoot featureGate that is not enabled")
+				}
+				tests.DisableFeatureGate(virtconfig.NonRoot)
+			})
+			AfterEach(func() {
+				tests.EnableFeatureGate(virtconfig.NonRoot)
+			})
+
+			It("should migrate root implementation to nonroot", func() {
```

Comment:
does this test need to be marked as "Serial" since it modifies feature gates?

Reply:
Migration tests already have it.